velox/vector/arrow/Bridge.cpp

```diff
@@ -39,11 +39,22 @@ static constexpr size_t kMaxBuffers{3};
 class VeloxToArrowBridgeHolder {
  public:
   VeloxToArrowBridgeHolder() {
```

Comment:
unrelated to your PR, but maybe we could just initialize buffers_ to null in the constructor to avoid having to spell this out?

```
std::vector<const void*> buffers_{numBuffers_, nullptr};
```